packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -864,22 +864,25 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	 */
 	public onReorderChange(column: StarkColumnSortChangedOutput): void {
 		if (column.sortable) {
-			this.resetSorting(column);
 			const sortedColumn = find(this.columns, { name: column.name });
+			const newOrderProperties: string[] = [];
 			if (sortedColumn) {
 				sortedColumn.sortPriority = 1;
 				switch (column.sortDirection) {
 					case "asc":
-						sortedColumn.sortDirection = "desc";
+						newOrderProperties.push("-" + sortedColumn.name);
 						break;
 					case "desc":
-						sortedColumn.sortDirection = "";
 						break;
 					default:
-						sortedColumn.sortDirection = "asc";
+						newOrderProperties.push(sortedColumn.name);
 						break;
 				}
 			}
+			this.orderProperties = newOrderProperties; // enforcing immutability :)
+			this.isMultiSorting = false;
+			this.cdRef.markForCheck();
```

Comment:
Why do we need `this.cdRef.markForCheck()` ?
It seems to work without this. Could you check ?